### ISSUE TITLE
split of target domain for  S→U and E→H evaluation

### ISSUE THREAD

**Issue by wlin-at** [ChenJinBIT/HPDA#2]
Hi Jin,

thanks for the refreshing work. I have a question regarding the split of target domain into training set and test set in the S→U and E→H evaluation. 
As UCF101 and HMDB51 both have 3 splits. Did you run the experiments 3 times and take the average results of the 3 splits? Or did you only use one of the split? 
I could not find any explanation of split for  S→U and E→H in the related work (HiGAN and SymGAN 2019). 
Looking forward to your reply and thanks in advance.

Best,
Wei

**Comment by ChenJinBIT** [comment]
Thanks for your attention to our work. I use the total of UCF101 and HMDB51 of corresponding common classes following the unsupervised domain adaptation setting. The annotation of the target domain is only used for evaluation.

**Comment by wlin-at** [comment]
> Thanks for your attention to our work. I use the total of UCF101 and HMDB51 of corresponding common classes following the unsupervised domain adaptation setting. The annotation of the target domain is only used for evaluation.

Thank you so much for the fast response. So there is no split of target domain into target train set and target test set? (e.g.  the official splits of HMDB51 always have 70 videos in training set and 30 videos in test set for each class). 
In your evaluation, the entire target data (e.g. 13 classes in HMDB51) is used during trainining. Then you test on the same target data (which are already used during training) like in a transductive setting? I notice that you are from the same group as the authors of HiGAN 2018 and SymGAN 2019 papers. Do you know if this is also the case (no split of target train and target test) for the evaluation in these two works?

I just want to make sure that I can have fair comparison by using the same evaluation setting. Please correct me if there is any misunderstanding. Thank you so much!

Best
Wei

**Comment by ChenJinBIT** [comment]
The train and test set of the target domain are the same, i.e., the whole dataset of common classes, and the difference between the training and test phase is that the annotation of video is only available during the test. We use the same setting of this paper to reproduce the results of HiGAN 2018 and SymGAN 2019 2019 for comparison. 

In HiGAN 2018 and SymGAN 2019 2019, using the official splits of the target data, the experiment was run three times, and the average results of the three splits were taken.
